### PR TITLE
HSC-24: Added a new vaccinationswidget

### PR DESCRIPTION
**Pull request by reagan-meant**
Removed the old vaccination widget.
Added a new vaccinations widget

### CODE REVIEW

File: configuration/openmrs/apps/customDisplayControl/views/immunization.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                    {{obs.name | translate }}
+                </span>
+            </td>
+            <td>
+                <span>
+                    {{obs.vaccationSequenceValue | translate }}
+                </span>
+            </td>
+            <td>
+                <span ng-class="{ 'is-abnormal': !obs.date}">
```
Comment (rbuisson) [inline configuration/openmrs/apps/customDisplayControl/views/immunization.html:41]: Let's just remove that formatting.

File: configuration/openmrs/i18n/clinical/locale_fr.json [hunk 2]
```diff
@@ -381,5 +381,10 @@
   "VISIT_TITLE_PATIENT_INFORMATION": "Informations patient",
   "VISIT_TITLE_RADIOLOGY_KEY": "Radiologie",
   "VISIT_TITLE_TREATMENTS_KEY": "Médicaments",
-  "YEARS_LABEL": "An(s)"
+  "YEARS_LABEL": "An(s)",
+  "VACCINATIONS": "Vaccinations",
+  "VACCINE": "Vaccin",
+  "SEQUENCE_NUMBER": "Numéro des vaccinations.",
```
Comment (rbuisson) [inline configuration/openmrs/i18n/clinical/locale_fr.json:387]: "Numéro de séquence"

**Review by rbuisson — CHANGES_REQUESTED** [review]
- Change the label of the widget to "Vaccination summary"
- Remove the 'red' color
- Append the sequence number to the vaccine name (for reading clarity)
- In summary view, show only the most recent sequence for a given vaccine
- In extended view, show all vaccination history (incl. manufacturer, lot number...)

File: configuration/openmrs/apps/customDisplayControl/views/immunization.html (hunk 1 above)
Comment (reagan-meant) [reply configuration/openmrs/apps/customDisplayControl/views/immunization.html]: done

File: configuration/openmrs/i18n/clinical/locale_fr.json (hunk 2 above)
Comment (reagan-meant) [reply configuration/openmrs/i18n/clinical/locale_fr.json]: done

**Pull request merged**